public/upload_images_to_server.rb

```diff
@@ -252,7 +252,7 @@ def upload_images_to_build_with_id(id, images_to_upload, test_image_folder)
 def compute_image_md5s_json(test_image_folder)
   image_md5s = {}
   traverse_directories(test_image_folder) do |ancestry, file|
-    md5 = `md5 -q #{file}`
+    md5 = Digest::MD5.file(file).hexdigest
```

Comment:
is this included in every default ruby installation or does it have to be explicitly installed?

Reply:
it's in the standard ruby library

Reply:
This is the same command we are using in the `test_image` model so this should be fine https://github.com/Workday/vizzy/blob/master/app/models/test_image.rb#L15